src/renderer/src/routes/Onboarding/DeviceNamingScreen.tsx

```diff
@@ -227,3 +223,7 @@ export function DeviceNamingScreenComponent() {
 		</Container>
 	)
 }
+
+export function getUtf8ByteLength(text: string): number {
+	return new TextEncoder().encode(text).length
+}
```

Comment:
maybe outside the scope of this PR but would eventually move this to something like`renderer/lib/text.ts`